<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProjectWidget</class>
 <widget class="QWidget" name="CreateProjectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>141</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>当前工作工程</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="currentProjectNameLabel">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>194</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="comboBox">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>采集</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>计算&amp;备份</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>实际与计划文件校正</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>删除从机数据（慎用）</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="importProjButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>导入工程</string>
     </property>
     <property name="icon">
      <iconset resource="../../resource/gui/icons.qrc">
       <normaloff>:/image/open-file.png</normaloff>:/image/open-file.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="createProjButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>创建新工程</string>
     </property>
     <property name="icon">
      <iconset resource="../../resource/gui/icons.qrc">
       <normaloff>:/image/new archive.png</normaloff>:/image/new archive.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="fileSynchronizationButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>文件同步</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource/gui/icons.qrc"/>
 </resources>
 <connections/>
</ui>
